pennylane_qiskit/qiskit_device2.py

```diff
@@ -289,7 +266,7 @@ def __init__(
             raise ValueError(f"Backend '{backend}' supports maximum {available_qubits} wires")
 
         self.reset()
-        self._update_kwargs()
+        # self._update_kwargs()
```

Comment:
Temporary. We come back to this in #547

Reply:
Better to remove it with a TODO rather than leaving it as a comment.